### ISSUE TITLE
Use healthchecks in deployer

### ISSUE THREAD

**Issue by alrevuelta** [waku-org/waku-simulator#36]
Use docker [healthchecks](https://docs.docker.com/reference/dockerfile/#healthcheck) in RLN contract deployer container, so that `nwaku` nodes don't start until RLN contract has been deployed.

Right now, we just use a `depends_on` in `nwaku` containers, which means that nwaku nodes start as soon as `contract-repo-deployer` starts and that wrong. We must wait until `contract-repo-deployer` has finished.

**Comment by stubbsta** [comment]
@alrevuelta I found another way to achieve the same objective.
Once the contracts have been deployed the `contract-repo-deployer` service stops running. It's possible to add a condition to the `depends on` field for the nwaku service in docker compose like this:
```
    depends_on:
      contract-repo-deployer:
          condition: service_completed_successfully
```
No other code is required.
The `nwaku` services are not created until the `contract-repo-deployer` service exits/stops successfully.

Do we specifically want to use the `healthchecks` approach or will this method be sufficient?

**Comment by alrevuelta** [comment]
@stubbsta Nice, that approach looks good. As long as we can block nwaku nodes to start before the contract has been deployed, I'm ok with any way to fix it.

**Comment by stubbsta** [comment]
This is address in PR https://github.com/waku-org/waku-simulator/pull/45

**Comment by stubbsta** [comment]
@alrevuelta I agree, I have reverted the docker-compose file to the default in PR https://github.com/waku-org/waku-simulator/pull/45 and I will  create a separate PR for just this issue

**Comment by alrevuelta** [comment]
Closed by https://github.com/waku-org/waku-simulator/pull/48
